lib/rialto/etl/service_client/web_of_science_client.rb

```diff
@@ -56,10 +56,12 @@ def perform_initial_query!
 
         def connect_with_retries(path:)
           response = client.get(path)
+          raise "#{response.reason_phrase}: #{response.status}  (#{response.body})" unless response.success?
           JSON.parse(response.body)
         rescue StandardError => exception
-          logger.warn "Error fetching #{path}: #{exception.message} (#{exception.class})"
-          {}
+          msg = "Error fetching #{path}: #{exception.message} (#{exception.class})"
+          logger.warn msg
+          raise msg
```

Comment:
Why don't we re-raise the original error, so that it's easier to track down the cause?

Reply:
I changed this so re-raises the original error.